lib/time.ml

```diff
@@ -24,6 +24,7 @@
  *)
 
 open Lwt
+[@@@warning "-3"]
```

Comment:
see above, imho we should fix warnings instead of disabling them module-globally